apps/website/src/pages/docs/badge.mdx

```diff
@@ -13,15 +15,25 @@ import LiveExample from '~/components/LiveExample.astro';
 
 <p>{frontmatter.description}</p>
 
-<LiveExample src={frontmatter.exampleCodeFile1} />
+<LiveExample src={frontmatter.exampleCodeMain} />
 
-Badges are visual identifiers that use data-visualization colors and a label to categorize certain components, such as project tiles. They are similar to tags in shape but they are not interchangeable, as they fulfill different roles. Unlike tags which are user-generated, badges are provided by the system.
+Badges are small visual identifiers with a label to categorize certain things, such as tiles or table rows. Badges are similar to [tags](tag) in shape but they are not interchangeable, as they fulfill different roles. Unlike tags which are user-generated, badges are provided by the system.
 
 ## Appearance
 
-Badges are colorful rectangles with rounded corners, with a label in white, all caps font on top. They are available in one size only; the length is flexible to accommodate different labels, but the height or font size cannot be increased. The font type and color remains the same no matter what.
+Badges are colorful rectangles with rounded corners, with a label in white, all caps font on top. Badges are available in one size only; the length is flexible to accommodate different labels, but the height or font size should not be increased. There are two different categories of colors to be used by badges:
```

Comment:
I have to ask, what is the purpose of these long descriptions about "appearance"?

We already show the demos so the text is redundant and will become outdated if we make visual changes.

Reply:
I've removed the Appearance description & renamed the section to `Types` to align with #997.

Reply:
nit: can you name it "variants"?

"type" is an overloaded term that can mean like 5 different things.

Reply:
Done, will make the change in dialog as well.